application/src/main/java/dev/aws101/registration/RegistrationService.java

```diff
@@ -15,7 +15,7 @@
   private final Set<String> validInvitationCodes;
 
   public RegistrationService(AWSCognitoIdentityProvider awsCognitoIdentityProvider,
-                             @Value("${TODO_COGNITO_USER_POOL_ID}") String userPoolId,
+                             @Value("${COGNITO_USER_POOL_ID}") String userPoolId,
```

Comment:
Removed the `TODO` prefix from all these env vars because I think the way we're handling them now is fine (i.e. passing them via CloudFormation instead of via Docker).